mesonbuild/compilers/c.py

```diff
@@ -430,6 +432,7 @@ def get_options(self) -> 'MutableKeyedOptionDictType':
         opts = super().get_options()
         opts.update({
             OptionKey('winlibs', machine=self.for_machine, lang=self.language): coredata.UserArrayOption(
+                'c_winlibs',
```

Comment:
Should this also use the create_option helper?

Reply:
oups! I missed it.